custom_components/elasticsearch/datastreams/index_template.json

```diff
@@ -82,7 +82,6 @@
                                             "type": "float"
                                         },
                                         "boolean": {
-                                            "ignore_malformed": true,
```

Comment:
Why is this no longer needed?

Reply:
Supported was only added in 8.8 :(

We don't technically need it for Boolean as we aren't taking raw values and stuffing them in this field so I don't think there's any way to actually end up with a malformed value